composer.json

```diff
@@ -7,24 +7,41 @@
     "license": "MIT",
     "authors": [
         {
-            "name": "Shogo Kawahara",
-            "email": "shogo.a.kawahara@mail.rakuten.com"
+            "name": "Andrew Miller",
+            "email": "andrew.miller@rakuten.com"
         }
     ],
     "require": {
-        "php": ">=5.2.3",
+        "php": ">=5.3.19",
```

Comment:
[5.5 or earlier versions have reached EoL](http://php.net/supported-versions.php). the version range should be `>= 5.6.0`.